KomaMRIBase/src/datatypes/simulation/DiscreteSequence.jl

```diff
@@ -79,17 +79,27 @@ based on simulation parameters.
 - `seq`: (`::Sequence`) sequence
 
 # Keywords
-- `sim_params`: (`::Dict{String, Any}`, `=default_sim_params()`) simulation parameter dictionary
+- `sampling_params`: (`::Dict{String, Any}`, `=default_sampling_params()`) sampling
+    parameter dictionary
 
 # Returns
 - `seqd`: (`::DiscreteSequence`) DiscreteSequence struct
 """
-function discretize(seq::Sequence; sim_params=default_sim_params())
-    t, Δt      = get_uniform_times(seq, sim_params["Δt"]; Δt_rf=sim_params["Δt_rf"])
+function discretize(seq::Sequence; sampling_params=default_sampling_params())
+    t, Δt      = get_uniform_times(seq, sampling_params["Δt"]; Δt_rf=sampling_params["Δt_rf"])
     B1, Δf     = get_rfs(seq, t)
     Gx, Gy, Gz = get_grads(seq, t)
     tadc       = get_adc_sampling_times(seq)
     ADCflag    = [any(tt .== tadc) for tt in t] #Displaced 1 dt, sig[i]=S(ti+dt)
     seqd       = DiscreteSequence(Gx, Gy, Gz, complex.(B1), Δf, ADCflag, t, Δt)
     return seqd
 end
+
+"""
+Returns a dictionary with default simulation parameters.
+"""
+function default_sampling_params(sampling_params=Dict{String,Any}())
```

Comment:
Use this in default_sim_params in Core

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/9f31465294b9130f9cefdaaa73efa3630eda2e82